product_docs/docs/migration_toolkit/55/02_supported_operating_systems_and_database_versions.mdx

```diff
@@ -9,8 +9,8 @@ title: "Supported platforms and databases"
 
 You can use the following database product versions with Migration Toolkit:
 
--   PostgreSQL versions 12, 13, 14, 15, and 16
--   EDB Postgres Advanced Server versions 12, 13, 14, 15, and 16
+-   PostgreSQL versions 12, 13, 14, 15, 16, and 17
+-   EDB Postgres Advanced Server versions 12, 13, 14, 15, 16, and 17
```

Comment:
Since we maintain last 5 major releases, EPAS 12 should be removed from the list.